### DISCUSSION TITLE
Bug i holdchat: Datomærkning mangler.

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#33] [Bugs] [answered]
Fra Malene:

Jeg oplever en bug, når jeg åbner en holdchat fra back-end. Ved de første to kommentarer står der dato og klokkeslæt for hvornår den er slået op, men det gør der ikke for de to sidste. 

![image](https://github.com/xooom-community/product/assets/167326849/384c8bf3-288a-4b4a-aca1-888ca6716b29)

**Comment by mikkelsl** [comment, marked as answer]
Har lavet en opgave her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=65525244
